packages/website/src/pages/FileAction.tsx

```diff
@@ -26,10 +26,10 @@ function FileAction() {
   // Support we have a folder, containing a shortcut to a README document,
   // the rInner is README and the rOuter is the folder.
   const { inMost: rInner, outMost: rOuter } = useRender();
-  const [revisionsEnabled, setRevisionsEnabled] = useState(false);
+  let [revisionsEnabled, setRevisionsEnabled] = useState(false);
   function toggleRevisions() {
```

Comment:
It should be:

```js
const toggleRevisions = useCallback(() => {
    setRevisionsEnabled(v => !v);
}, []);

const commandBarItems: ICommandBarItemProps[] = useMemo(() => {
  ......
  onClick: () => { toggleRevisions(); },
}, [...., toggleRevisions])
```

Reply:
This doesn't work in my testing. That is, the modifications still show when I click on a new document.

Reply:
I think they are different problems. You need to reset the `revisionsEnabled` to `false` when `rInner` or `rOuter` is changed. This is because the FileAction component is not unmounted when click on a new document.

Reply:
I pushed a commit which does this. It still doesn't seem like the right way, but like 2 other versions of this PR it works when tested.